govuk-one-login/authentication-api
# AUT-2435: Enable snapstart in authdev1 for testing

## What

Wire up all API components so they can be configured to use SnapStart.

Turn it on across the board for authdev1 for regression testing

## How to review

- Code review

1. deploy the branch to authdev1, observe that snapstart is enabled for all lambdas
2. deploy the branch to authdev2, observe that nothing changes

## Related PRs

Depends on https://github.com/govuk-one-login/authentication-api/pull/5644. I don't want to make these changes multiple times.
